changelog.d/20241112_110807_phil-PL-133007_nginx-dheat-mitigation_scriv.md

```diff
@@ -0,0 +1,22 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+- possibly impacts client connectivity to Nginx. Semi-official connectivity testers showed no change for popular clients and/or libraries but cannot cover every single implementation out there
```

Comment:
I'd like to adapt this. To non-technical people this sounds worse than it actually is IMHO.

Suggestion:

- There is a small, but non-zero potential that some exotic clients may experience connectivity issues with nginx. Multiple connectivity testing tools showed no change for clients and/or libraries but cannot cover every single implementation out there.

Reply:
Thanks for the suggestion!